config/routes.rb

```diff
@@ -37,7 +37,8 @@
   get '/video', to: 'video#show'
 
   get "/auth/:provider/callback" => "github_sessions#create"
-  get "/signout" => "github_sessions#destroy", :as => :signout
+  get "/auth/failure", to: "github_sessions#create"
+  get "/signout" => "sessions_controller#destroy", :as => :signout
```

Comment:
Signout now uses the sessions_controller

Reply:
Awesome!